src/components/RoomSelector.vue

```diff
@@ -21,7 +21,7 @@
   -->
 
 <template>
-	<NcModal size="normal"
```

Comment:
These changes broke usage of component (and tests for it) in too many places:
- Deck, Maps, Search, Collections integrations
- Talk inner features

I'd suggest you to drop this commit, so we could also work on it separately

Talk | Search | Deck
--|--|--
![image](https://github.com/nextcloud/spreed/assets/93392545/2c00b634-1d5c-4d27-adc3-29c11232d3d2) | ![image](https://github.com/nextcloud/spreed/assets/93392545/8b3a5e0a-45ad-49f7-ad9b-603aa3d466c7) | ![image](https://github.com/nextcloud/spreed/assets/93392545/4ddf832b-2a1e-4944-80c6-e0467403f067)